api/products/productController.js

```diff
@@ -0,0 +1,69 @@
+const productService = require("./productService");
+
+exports.productPost = (req, res, next) => {
+  let bodyData = req.body;
+  bodyData.alias = bodyData.title.split(" ").join("-").toLowerCase();
+  productService
+    .createProduct(bodyData)
+    .then((data) => {
+      res.status(201).json({ message: "created", data });
+    })
+    .catch((err) => {
+      next(err);
+    });
+};
+
+exports.productList = (req, res, next) => {
```

Comment:
We need to think about more filters. Examples 
* Based on price higher, lower or b/w the ranges.
* We need to think about product brand or company as well, which is missing in product schema.

But this can go in an another feature.